app-store-connect/src/bundle_api.rs

```diff
@@ -0,0 +1,123 @@
+use crate::{AppStoreConnectClient, Result};
+use serde::{Deserialize, Serialize};
+
+const APPLE_CERTIFICATE_URL: &'static str = "https://api.appstoreconnect.apple.com/v1/bundleIds";
+
+impl AppStoreConnectClient {
+    pub fn register_bundle_id(&self, identifier: &str, name: &str) -> Result<BundleIdResponse> {
```

Comment:
Since we now have discrete components of the client, what do you think about having a common string prefix to group common functions? e.g. `bundle_*` for the bundle API?

Alternatively, we could group things in traits and require people to bring the trait in scope to make the APIs callable.

Reply:
well I don't think we should have any conflicts. other than create/register variation, they match the REST style well, with consistent naming `VERB_OBJECT`. So they have a common postfix instead which is more idiomatic in rust. Although not completely as `get_` is usually omitted.